pkg/commands/options/build.go

```diff
@@ -24,11 +24,15 @@ import (
 type BuildOptions struct {
 	ConcurrentBuilds     int
 	DisableOptimizations bool
+	Platform             string
 }
 
 func AddBuildOptions(cmd *cobra.Command, bo *BuildOptions) {
 	cmd.Flags().IntVarP(&bo.ConcurrentBuilds, "jobs", "j", runtime.GOMAXPROCS(0),
 		"The maximum number of concurrent builds")
 	cmd.Flags().BoolVar(&bo.DisableOptimizations, "disable-optimizations", bo.DisableOptimizations,
 		"Disable optimizations when building Go code. Useful when you want to interactively debug the created container.")
+	cmd.Flags().StringVar(&bo.Platform, "platform", "linux/amd64",
+		"Which platform to use when pulling a multi-platform base. Format: all | <os>[/<arch>[/<variant>]]")
+
```

Comment:
Isn't this technically a breaking change if folks are using `GOOS/GOARCH` with `ko` today?

Reply:
Done (see pkg/commands/config.go)